### PR TITLE
Stage 1 experimental changes for RFC 0012 - orchestrator fieldset

### PR DESCRIPTION
**Pull request by ebeahan**
Adds `orchestrator.*` fieldset from [RFC 0012](https://github.com/elastic/ecs/blob/master/rfcs/text/0012-orchestrator-field-set.md) (https://github.com/elastic/ecs/pull/1230) to the experimental schema and artifacts.

### CODE REVIEW

File: experimental/schemas/orchestrator.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      level: extended
+      type: keyword
+      description: >
+        Name of the cluster.
+
+    - name: cluster.url
+      level: extended
+      type: keyword
+      description: >
+        URL of the cluster.
```
Comment (jsoriano) [inline experimental/schemas/orchestrator.yml:21]: Perhaps this description should be specific about being the URL of the API used to manage the cluster.
Something like this.
```suggestion
        URL of the API used to manage the cluster.
```

**Review by jsoriano — APPROVED** [review]
Thanks! Added a small suggestion about the docs of a field, not sure if this can be applied now after the RFC.

File: experimental/generated/beats/fields.ecs.yml [hunk 2]
```diff
@@ -4120,6 +4120,74 @@
...
+      type: keyword
+      ignore_above: 1024
+      description: Namespace in which the action is taking place.
+      example: kube-system
+      default_field: false
+    - name: organization
+      level: extended
+      type: keyword
+      ignore_above: 1024
+      description: Organization affected by the event (for multi-tenant orchestrator
```
Comment (kaiyan-sheng) [inline experimental/generated/beats/fields.ecs.yml:4166]:
```suggestion
      description: Organization affected by the event (for multi-tenant orchestrator setups).
```

**Review by kaiyan-sheng — APPROVED** [review]
lgtm, besides one nit :)

File: experimental/schemas/orchestrator.yml (hunk 1 above)
Comment (ebeahan) [reply experimental/schemas/orchestrator.yml:21]: Thanks, great suggestion!

Small adjustments to improve descriptions are fine, as long as we're not significantly changing the intent of the field.

**Pull request merged**